package.json

```diff
@@ -22,29 +26,34 @@
     "@angular/platform-browser": "^6.0.3",
     "@angular/platform-browser-dynamic": "^6.0.3",
     "@angular/router": "^6.0.3",
+    "concurrently": "^3.6.0",
     "core-js": "^2.5.4",
+    "json-concat": "0.0.1",
+    "json-server": "^0.14.0",
```

Comment:
Some of these new additions should really be dev-dependencies.

Reply:
good point!